deploy/helm/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [2.7.0-alpha.1] - 2023-07-17
```

Comment:
Nitpick - This probably doesn't belong here.

Reply:
That is true, I will remove it when merging this branch to master after it gets released